pallets/dapp-staking-v3/src/benchmarking/utils.rs

```diff
@@ -49,34 +49,88 @@ pub(super) fn advance_to_era<T: Config>(era: EraNumber) {
     }
 }
 
+/// Advance blocks until the specified era has been reached.
+///
+/// Relies on the `force` approach to advance one era per block.
+pub(super) fn force_advance_to_era<T: Config>(era: EraNumber) {
+    assert!(era >= ActiveProtocolState::<T>::get().era);
+    while ActiveProtocolState::<T>::get().era < era {
+        assert_ok!(DappStaking::<T>::force(
+            RawOrigin::Root.into(),
+            ForcingType::Era
+        ));
+        run_for_blocks::<T>(One::one());
+    }
+}
+
 /// Advance blocks until next era has been reached.
-pub(super) fn advance_to_next_era<T: Config>() {
+pub(super) fn _advance_to_next_era<T: Config>() {
```

Comment:
Is it better to just remove it as it's dead code? and line 83,99 and all other

Reply:
I'll remove it later, let me see first how the code will evolve in the next month 😅